src/main/flight/mixer.c

```diff
@@ -766,17 +800,59 @@ float applyThrottleLimit(float throttle) {
     return throttle;
 }
 
-FAST_CODE_NOINLINE void mixTable(timeUs_t currentTimeUs) {
+FAST_RAM_ZERO_INIT float lastRcDeflection[3], stickMovement[3];
+
+void applyAirMode(float *motorMix, float motorMixMax) {
+    float motorMixNormalizationFactor = motorMixRange > 1.0f && hardwareMotorType != MOTOR_BRUSHED ? motorMixRange : 1.0f;
+    float motorMixDelta = 0.5f * motorMixRange;
+    float authorityZeroThrottle, authorityFullThrottle;
+
+    if (isAirmodeActive()) {
```

Comment:
This is a bit contradicting to the name of the function. In here applyAirMode() is called, with still inside check if airmode is active.
In other words. applyAirMode functions does everything. AIRMODE OFF, AIRMODE ON and AIRMODE 2.0

Reply:
I agree, we have to change the name of the method "applyAirMode" into something more correct. Any idea?

Reply:
I preferred refactoring the logic a little, at least now we don't call that method inside here.
The problem still remains, because the "airMode" logic gets always applied, even if the "isAirmodeActive" flag isn't active (but also the old logic did).
It's something like isAirmodeActive should be called useFullAuthority or something like that.

Reply:
well, to be fair this code always forces you to have some level of airmode present even when airmode is off, but perhaps calling it applyMixerClipFix()??? don't quite like the name but it describes it fairly well.

Reply:
Yes, something like that, it prevents clipping but I wouldn't call it "fix", maybe adjustment.